src/Calculator/Views/Memory.xaml.cpp

```diff
@@ -135,7 +112,6 @@ void Memory::MemoryList_Unloaded(_In_ Object^ sender, _In_ RoutedEventArgs^ e)
 
 MemoryItemViewModel^ Memory::GetMemoryItemForCurrentFlyout()
 {
-    auto listViewItem = m_memoryItemFlyout->Target;
-
+    auto listViewItem = MemoryContextMenu->Target;
     return safe_cast<MemoryItemViewModel^>(MemoryListView->ItemFromContainer(listViewItem));
```

Comment:
Same feedback here safe_cast would throw so the checks for nullptr in the calling code would not occur.